Sources/XCTest/XCTestCase.swift

```diff
@@ -26,46 +58,46 @@ extension XCTestCase {
             // TODO: When using the Objective-C runtime, XCTest is able to throw an exception from an assert and then catch it at the frame above the test method. This enables the framework to effectively stop all execution in the current test. There is no such facility in Swift. Until we figure out how to get a compatible behavior, we have decided to hard-code the value of 'true' for continue after failure.
         }
     }
-    
-    public func invokeTest() {
```

Comment:
Thanks for pointing that out, @modocache, I hadn't considered that method being a part of the XCTestCase API. Having looked at the docs now though, it seems that  the current semantics of this method (run all the test methods in the test case) are significantly different than that of Xcode XCTest's (run a single test method). I would thus look at this as removing a potential point of confusion until such time as proper API compatibility may be attempted.